models/Tool/SettingsStore.php

```diff
@@ -24,7 +24,17 @@
  */
 final class SettingsStore extends Model\AbstractModel
 {
-    protected static array $allowedTypes = ['bool', 'int', 'float', 'string'];
+    public const TYPE_BOOLEAN = 'bool';
+    public const TYPE_FLOAT = 'float';
+    public const TYPE_INTEGER = 'int';
+    public const TYPE_STRING = 'string';
+
+    protected static $allowedTypes = [
```

Comment:
Don't remove type hint
```suggestion
    protected static array $allowedTypes = [
```

Reply:
Thanks, didn't notice I did so.

Reply:
If we don't change this to an enum: I think this should be a `const` instead.